docs/installation_and_tutorial.md

```diff
@@ -343,32 +260,43 @@ Finally the logs are shown in the runtime:
 
 ## Trends
 
-To add a trend go to the add screen items and in the advanced bar select trend control:
+To add a trend go to the Toolbox, select trend control and drag and drop into the screen:
 
-![trends1](graphics/trends1_new.jpg)
+![trends1](graphics/trends1.png)
 
-In its properties to add different trends to appear in the item, go to Trends:
+In its properties to add different trends to appear in the item, go to Trends and select the tag/logging tag you want to control::
 
 ![trends2](graphics/trends2.png)
 
-And select the tag you want to control:
-
-![trends3](graphics/trends3.png)
-
 In the runtime the trend will be filled:
 
 ![trends4](graphics/trends4.png)
 
-## Start the runtime
+## TIA Portal guidelines
+
```

Comment:
I would also add the SIOS-reference /SIOS link to the user manual of WinCC Unified Engineerign